tests/virt_autotest/uefi_guest_verification.pm

```diff
@@ -89,8 +95,20 @@ sub check_guest_pmsuspend_enabled {
 
     $self->do_guest_pmsuspend($_, 'mem') foreach (keys %virt_autotest::common::guests);
     if (is_kvm_host) {
-        $self->do_guest_pmsuspend($_, 'hybrid') foreach (keys %virt_autotest::common::guests);
-        $self->do_guest_pmsuspend($_, 'disk')   foreach (keys %virt_autotest::common::guests);
+        foreach (keys %virt_autotest::common::guests) {
+            if (is_sle('=15-sp4') and ($_ =~ /12-sp5/img)) {
```

Comment:
With this if condition, it seems 15sp3 supports the feature while 15sp4 does not? It look weird, generally newer products supports more than older, so the reverse are more reasonable to happen.

Reply:
I think this is mainly caused by 12-SP5 guest, not 15-SP4 host. There is no mechanism for the guest to perform "power action" if being suspended from outside.

Reply:
I see. How about change "is_sle('=15-sp4')" to match "developing host", then you won't need to change test code for 15sp5(if still not supported yet), until this behavior becomes supported on "some" new product?

Reply:
Matching developing host version does not cover backwardly. The easiest way is is_sle('>=15') I think.

Reply:
Either way works. By the way, we will likely not to have old host with 12sp5 as guest in new product's LTP.

Reply:
is_sle('>=15') covers more, so it also better to cope with code change. Done.